BackgroundWorker/BackgroundWorker.csproj

```diff
@@ -10,10 +10,10 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Storage.Blobs" Version="12.21.2" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR" Version="1.0.4" />
```

Comment:
This package is obsoleted. Now this logic is [part](https://learn.microsoft.com/en-us/answers/questions/958531/the-package-microsoft-aspnetcore-signalr-is-deprec) of .net. This package should be removed and it still should work

Reply:
> This package is obsoleted. Now this logic is [part](https://learn.microsoft.com/en-us/answers/questions/958531/the-package-microsoft-aspnetcore-signalr-is-deprec) of .net. This package should be removed and it still should work

The package version 1.0.10 is absolute we can consider 1.0.4

Reply:
As I said, this is not needed since now it's part of asp.net